Cargo.toml

```diff
@@ -15,4 +16,12 @@ pe = []
 vm-memory = ">=0.2.0"
 
 [dev-dependencies]
+criterion = "=0.3.0"
 vm-memory = {features = ["backend-mmap"]}
+
```

Comment:
I think it's useful to enable more optimizations than the default for the `[bench]` profile, so measurements are closer to what ppl do (or should do :D) for production builds. Seems like a good idea to enable `lto` at the very least, and potentially set `codegen-units` to `1` as well, i.e.:

```rust
[profile.bench]
lto = true
codegen-units = 1
```